appium/webdriver/extensions/keyboard.py

```diff
@@ -129,3 +129,9 @@ def _addCommands(self):
             ('POST', '/session/$sessionId/appium/device/press_keycode')
         self.command_executor._commands[Command.LONG_PRESS_KEYCODE] = \
             ('POST', '/session/$sessionId/appium/device/long_press_keycode')
+        self.command_executor._commands[Command.SET_IMMEDIATE_VALUE] = \
```

Comment:
Ah...
SET_IMMEDIATE_VALUE, REPLACE_KEYS and CLEAR are referred by WebElement.py.
Could you move these three items into like `appium/webdriver/extensions/webelement.py` to separate them?
And LOCATION_IN_VIEW as well.

Reply:
Okay, I just followed per your latest comment on the issue.
Will change the API's and tests again.
SET_IMMEDIATE_VALUE, REPLACE_KEYS, CLEAR and LOCATION_IN_VIEW to web element.py

Reply:
done! pls check